packages/suite/src/hooks/wallet/sign-verify/useSignAddressOptions.ts

```diff
@@ -0,0 +1,104 @@
+import { useMemo } from 'react';
+import { useTranslation } from '@suite-hooks';
+import type { Account } from '@wallet-types';
+import type { ExtendedMessageDescriptor } from '@suite-types';
+import type { State as RevealedAddresses } from '@wallet-reducers/receiveReducer';
+
+export type AddressItem = {
+    label: string;
+    value: string;
+};
+
+export const useSignAddressOptions = (
+    account: Account | undefined,
+    revealedAddresses: RevealedAddresses,
+) => {
+    const reduceAddresses = (
+        addresses: { address: string; path: string }[],
+        category: ExtendedMessageDescriptor['id'],
+    ) =>
+        addresses.reduce(
+            (prev, { address, path }) => ({
+                ...prev,
+                [path]: {
+                    path,
+                    address,
+                    category,
+                },
+            }),
+            {},
+        );
+
+    const signAddresses = useMemo(() => {
+        switch (account?.networkType) {
+            case 'bitcoin':
+                return {
+                    ...reduceAddresses(
+                        revealedAddresses.length
+                            ? revealedAddresses
+                            : (account.addresses?.unused || []).slice(0, 1),
+                        'TR_ADDRESSES_FRESH',
```

Comment:
nit: could be singular `Fresh address`

Reply:
If you reveal more addresses in Receive form before, you can then sign with any of them:
![image](https://user-images.githubusercontent.com/26326960/135451143-ea6cc83e-fa65-42ce-8fb1-42e1e068f0f9.png)